TombEngine/Scripting/Internal/TEN/Effects/EffectsFunctions.cpp

```diff
@@ -475,6 +475,24 @@ namespace TEN::Scripting::Effects
 		SpawnDynamicSpotLight(pos.ToVector3(), dir.ToVector3(), color, rad, fallOff, dist, ValueOr<bool>(castShadows, false), GetHash(ValueOr<std::string>(name, std::string())));
 	}
 
+	/// Emit dynamic fogbulb that lasts for a single frame.
+	// If you want a fogbulb that sticks around, you must call this each frame.
+	// @function EmitFxFog
+	// @tparam Vec3 pos position of the fogbulb
+	// @tparam[opt] int radius measured in "clicks" or 256 world units (default 20)
+	// @tparam[opt] float density from 0.0f to 1.0f (default 1.0f)
+	// @tparam[opt] Color color (default Color(255, 255, 255))
+	// @tparam[opt] string name if provided, engine will interpolate this light for high framerate mode (be careful not to use same name for different fogbulbs)
+	static void EmitFxFog(Vec3 pos, TypeOrNil<int> radius, TypeOrNil<float> density, TypeOrNil<ScriptColor> col, TypeOrNil<std::string> name)
```

Comment:
Also please rename to `EmitFogBulb` here.

Reply:
fixed